terraform/oas.yaml

```diff
@@ -22,7 +22,22 @@ paths:
               schema:
                 type: object
 
-  /event/{id}:
+  /Immunization/{id}:
+    put:
+      x-amazon-apigateway-integration:
+        uri: "${update_event.lambda_arn}"
+        payloadFormatVersion: "1.0"
+        passthroughBehavior: "when_no_match"
+        httpMethod: "PUT"
+        timeoutInMillis: 30000
+        type: "AWS_PROXY"
+      responses:
+        '200':
```

Comment:
Does this need the 201 as well?

Reply:
Oh I forgot about `oas.yaml`. That's not the only problem. I fixed everything that was missing in there. Removing `content` and also adding `Location` header. Thanks 🚀